### ISSUE TITLE
schema needs update

### ISSUE THREAD

**Issue by ashokgkumar** [jpalomo/kafka-connect-mssqlserver-cdc#7]
Hi John, since it is now pulling from the CT table directly, the select includes end_lsn and command_id.  You may need to update the pointer COLUMN_START to 7.  Also, the _command_id is something that isn't part of the base table, so I would rather remove it.  resultsMetadata.getColumnCount()-1 on the convertValueschema and convertValueRecord should do it.

**Comment by jpalomo** [comment]
@ashokgkumar So I started looking at this and then realized I could make a change that would avoid having to query the underlying CT tables directly that we discussed in [Issue 2](https://github.com/jpalomo/kafka-connect-mssqlserver-cdc/issues/2).  This change would avoid us having to make the above change.  I want to get your opinion on it before I go ahead and commit .  

I will put out a PR for later this evening or tomorrow, but essentially the change would be as follows:

1.   Updating the offsets that are to be used and instead of using the databaseName and the source table name as found in the [PartitionUtilities](https://github.com/jpalomo/kafka-connect-mssqlserver-cdc/blob/master/connect/src/main/java/com/github/jpalomo/connect/cdc/mssqlserver/util/PartitionUtilities.java#L30), I would concatenate the databaseName and the cdc capture instance object id.  The object id resolves to a unique integer within the database.

2.  Before executing the query to poll for the changes, check for the object and ensure it exists.  If it does not exist, a ConnectException will be thrown and the task will fail.  At this point, the task will remain in the FAILED state until manually restarted.  When the task gets restarted, it will get the new object id and thus the offset will not exist and it will begin querying from the beginning against that new capture instance (if it has the same name as the original instance that was disable). 

I think this might be a better approached.  It avoids having to manually reset offsets and yet provides some visibility in to operations occurring at the database that can impact connector.
